src/client/training.py

```diff
@@ -11,54 +11,66 @@
+            Take into account loading a page with a task suite and sending responses to the server. It is recommended that you set at least 60 seconds.
+        mix_tasks_in_creation_order:
+            * True — Tasks are grouped in suites in the order they were created.
+            * False — Tasks are chosen for a task suite in a random order.
+
+            Default: True.
+        shuffle_tasks_in_task_suite:
+            * True — Tasks from a task suite are shuffled on the page.
+            * False — Tasks from a task suite are placed on the page in the order they were created.
+
+            Default: True.
+        training_tasks_in_task_suite_count: The number of training tasks in one task suite.
+        task_suites_required_to_pass: The number of task suites that must be completed by a Toloker to get a training skill.
+        retry_training_after_days: The training can be completed again after the specified number of days to update the training skill.
+            If the parameter is not specified, then the training skill is issued for an unlimited time.
+        inherited_instructions:
+            * True — Project instructions are used in the training.
+            * False — Instruction, specified in the `public_instructions` parameter, are used.
+
+            Default: False.
+        public_instructions: Instructions for Tolokers used when the `inherited_instructions` parameter is False. Describe in the instructions how to complete training tasks.
+            You can use HTML markup inside `public_instructions`.
+        metadata: A dictionary with metadata.
+        owner: A training owner.
+        id: The ID of the training. Read only.
+        status: A training status. Read only.
+        last_close_reason: A reason why the training was closed last time. Read only.
+        created: The UTC date and time when the training was created. Read only.
+        last_started: The UTC date and time when the training was started last time. Read only.
+        last_stopped: The UTC date and time when the training was stopped last time. Read only.
     """
 
     @unique
     class CloseReason(ExtendableStrEnum):
-        """The reason for closing the pool the last time:
+        """A reason for closing a training.
 
         Attributes:
-            MANUAL: Closed by the requester.
-            EXPIRED: Reached the date and time set in will_expire.
-            COMPLETED: Closed automatically because all the pool tasks were completed.
-            NOT_ENOUGH_BALANCE: Closed automatically because the Toloka account ran out of funds.
-            ASSIGNMENTS_LIMIT_EXCEEDED: Closed automatically because it exceeded the limit on assigned task suites
-                (maximum of 2 million).
-            BLOCKED: Closed automatically because the requester's account was blocked by a Toloka administrator.
+            MANUAL: A training was closed by a requester.
+            EXPIRED: The lifetime of the linked pool has expired.
```

Comment:
Reached out to backend team to clarify about EXPIRED and NOT_ENOUGH_BALANCE statuses:
New training will not have these statuses so these are either lagecy statuses or a mistake in the old version of API documentation.
Can you please update these docstrings for one more time?

Reply:
Removed EXPIRED and NOT_ENOUGH_BALANCE statuses from docstring.
(Did you mean that?)

Reply:
@alexdrydew 
Checks passed. )